### ISSUE TITLE
Recommendations app though disabled still pushes annoying popups on client

### ISSUE THREAD

**Issue by the-moog** [nextcloud/server#20524]
### Steps to reproduce
1. Reboot client machine

### Expected behaviour
No popups

### Actual behaviour
Dozens of App Recommendations popups, I connect to three NC servers so it's very annoying and goes on for about 3 minutes.

### Server configuration

**Operating system:**
Ubuntu 18.04.04 LTS

**Web server:**
Apache 2.4.29

**Database:**
PostgresSQL 10.12

**PHP version:**
7.2.24

**Nextcloud version:** (see Nextcloud admin page)
18.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated for several years

**Where did you install Nextcloud from:**
Too long ago to recall

**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

Enabled:
  - accessibility: 1.4.0
  - activity: 2.11.0
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - dav: 1.14.0
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_external: 1.9.0
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - password_policy: 1.8.0
  - photos: 1.0.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - support: 1.1.0
  - survey_client: 1.6.0
  - systemtags: 1.8.0
  - text: 2.0.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - viewer: 1.2.0
  - workflowengine: 2.0.0
Disabled:
  - admin_audit
  - encryption
  - external
  - nextcloud_announcements
  - recommendations
  - user_ldap

</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
<?php
$CONFIG = array (
  'passwordsalt' => '<redacted>',
  'secret' => '<redacted>',
  'trusted_domains' =>
  array (
    0 => 'server.domain.com',
  ),
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'datadirectory' => '/opt/nextcloud/data',
  'overwrite.cli.url' => 'http://localhost',
  'dbtype' => 'pgsql',
  'version' => '18.0.3.0',
  'dbname' => 'nextcloud',
  'dbhost' => 'localhost',
  'dbport' => '',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'nextcloud',
  'dbpassword' => '<redacted>',
  'instanceid' => '<redacted>',
  'installed' => true,
  'maintenance' => false,
  'theme' => '',
  'loglevel' => 2,
  'data-fingerprint' => '<redacted>',
  'mail_domain' => 'domain.com',
  'mail_from_address' => 'nextcloud',
  'mail_smtpmode' => 'smtp',
  'mail_sendmailmode' => 'smtp',
  'mail_smtphost' => 'localhost',
  'trashbin_retention_obligation' => 'auto, 60',
  'updater.release.channel' => 'stable',
);

```
</details>

**Are you using external storage, if yes which one:** YES:local

**Are you using encryption:** No

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:**
Firefox latest

**Client**
2.6.4stable-Win64 (build 20200303)

**Operating system:**
Win10 (multiple systems)

### Logs

Logs will be posted only if requested and relivant to this issue

**Comment by kesselb** [comment]
https://github.com/nextcloud/recommendations would be the repository to report issues with the recommendations app. But the recommended apps notification is emitted somewhere else and not by the recommendations app. You probably need to mark the notification as read. Not sure if that is possible with the client. 

cc @nextcloud/server-triage 🤷‍♂️

**Comment by MorrisJobke** [comment]
I guess this is because the client shows all notifications again on every start. @nextcloud/desktop

**Comment by the-moog** [comment]
It's not the first run wizzard. It's the recommendations app which is enable by default.
This is far from the first run, it's been running for many years since early Owncloud days.

I did not know they requried marking as read in the client, even though the app which generated them is disabled.
I've done that and it's gone away on my two machines ( so in my case 6 sets of messages )

The problem is I'm an admin on several NC servers and on a bunch of workstations.  That means I have to acknowledge them on every workstation that I have logged in on if I did so before disabling recommendations.

Suggestion for improvement:
At startup, NC client should query the server and check the source of any pending messages, by getting a list of installed apps. If that source/app is disabled or other push mechanism turned off then the messages should be deleted instead of bothering the users.
